pkg/util/crdutil.go

```diff
@@ -296,6 +296,24 @@ var (
 		Type:     "date",
 		JSONPath: ".metadata.creationTimestamp",
 	}
+	tidbMonitorAdditionalPrinterColumns      []extensionsobj.CustomResourceColumnDefinition
+	readyTiDBMonitorAdditionalPrinterColumns = extensionsobj.CustomResourceColumnDefinition{
+		Name:     "READY",
+		Type:     "string",
+		JSONPath: `.status.statefulSet.conditions[?(@.type=="Ready")].status`,
+	}
+
+	tidbMonitorDesireColumn = extensionsobj.CustomResourceColumnDefinition{
+		Name:        "Desire",
+		Type:        "integer",
+		Description: "The desired replicas number of TiKV cluster",
```

Comment:
not `TiKV cluster` here.

Reply:
I ignored this in the process of repeatedly optimizing the code, sorry.  I have changed it to the right way.